var/spack/repos/builtin/packages/fzf/package.py

```diff
@@ -37,7 +35,9 @@ class Fzf(MakefilePackage):
 
     @classmethod
     def determine_version(cls, exe):
-        return Executable(exe)('--version', output=str, error=str).rstrip()
```

Comment:
Is this style no longer recommended? I have several other packages that do things like this like `gdal`.

Reply:
I wouldn't make it a hard request, but there's the risk of malformed versions. `fzf` for instance has the format `X.Y.Z` on some platforms and `X.Y.Z <random-text>` on other platforms.

Reply:
Okay, I'll leave other packages as is for now and wait until someone finds a weird version string before updating.